code-challenges.test.js

```diff
@@ -15,16 +15,35 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-// Example input: 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+describe('Fibonacci', () => {
+    it(' Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.', () => {
+        // Example input: 6
+        // Expected output: [1, 1, 2, 3, 5, 8]
 
-// Example input: 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+        // Example input: 10
+        // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+
+        expect(Fibonacci(6)).toEqual([1, 1, 2, 3, 5, 8])
+        expect(Fibonacci(10)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+})
 
 
 // b) Create the function that makes the test pass.
 
 
+function Fibonacci(n) {
+    // declare the array starting with the first 2 values of the fibonacci sequence
+    // starting at array index 1, and push current index 
+    // Add the previous index to the array
+
+    for (var fibonacciV = [0, 1], i = 1; i < n; i++)
+        fibonacciV.push(fibonacciV[i] + fibonacciV[i - 1])
+
+    return fibonacciV
+}
```

Comment:
A for loop is definitely a great approach to this problem!  There may be a few syntax things to fix, also make sure to name your function in camelCase.  I would recommend running the test and seeing it's outcome (and pseudocoding what your results are in here, even if it doesn't pass).  When a test fails, it will show you what the outcome is versus what you expect and can be very helpful in finding where there are problems.